### ISSUE TITLE
Weather app icon is not full-size

### ISSUE THREAD

**Issue by Aransh** [midnightchip/Asteroid#71]
As you can see in the screenshot, when tapping the weather app you can see the icon is not full sized, iPhone XS Max - iOS 12.1

![B5956C63-C7A1-4817-803E-6EBD0038F2B7](https://user-images.githubusercontent.com/11724502/57188808-379ffa80-6f0d-11e9-8562-a278946312d9.jpeg)

**Comment by the-casle** [comment]
Are you using any tweaks to modify the icons? It masks to snowboard

**Comment by Aransh** [comment]
No, no other tweak that affects icons except LiveRings but I don't believe this could relate
